crates/amaru/Cargo.toml

```diff
@@ -28,6 +28,8 @@ minicbor.workspace = true
 opentelemetry-otlp.workspace = true
 opentelemetry.workspace = true
 opentelemetry_sdk.workspace = true
+pallas-addresses.workspace = true
+pallas-codec.workspace = true
```

Comment:
Aren't those only test dependencies? Also, I believe they aren't needed because everything is in the kernel already. If they aren't, then re-export them from the kernel instead.